src/pages/home/report/ReportActionCompose/Suggestions.js

```diff
@@ -111,6 +117,29 @@ function Suggestions({
         [onSelectionChange, resetSuggestions, setShouldBlockSuggestionCalc, triggerHotkeyActions, updateShouldShowSuggestionMenuToFalse, getSuggestions],
     );
 
+    useEffect(() => {
+        if (prevIsDraggingOver === isDraggingOver) {
+            return;
+        }
+        if (isDraggingOver) {
+            updateShouldShowSuggestionMenuToFalse();
+        }
+    }, [isDraggingOver, prevIsDraggingOver, updateShouldShowSuggestionMenuToFalse]);
+
+    useEffect(() => {
+        if (![CONST.PLATFORM.WEB, CONST.PLATFORM.DESKTOP].includes(getPlatform())) {
+            return;
+        }
+        // hide popover on drag enter
+        const listener = () => {
+            updateShouldShowSuggestionMenuToFalse();
+        };
+        document.addEventListener('dragenter', listener);
+        return () => {
+            document.removeEventListener('dragenter', listener);
+        };
+    }, [updateShouldShowSuggestionMenuToFalse]);
```

Comment:
Could you explain how this block correlates to the one above it? Why do we need both?

Reply:
So, in other words...

- dragging a file _directly_ over the suggestions component **didn't** work before
- and after this PR, it starts working

Again, this is an unrelated change, as I see it.

Reply:
> I think with this Pr we are aiming to close popver/suggestion if we dragover a file in app

First and formost, this PR has to _fix the issue_. I approved the idea to fix the issue by closing popovers on drag-enter, but it was still done in the context of _the issue_.

The issue is this visual glitch:

![image](https://github.com/Expensify/App/assets/2590174/4e7d72fc-67be-41be-8b25-e659f174ab68)

The app entered the "drag state" (showing "Drag to upload" text, adding a "glass", being ready for drop, etc.), while we have a **dangling UI element** on top of it.

If the app doesn't enter the "drag state" on `main` right now, I can't see why we were obligated to make it enter the "drag state" after this PR.

Does this make sense?

Reply:
While the added code is, arguably, some improvement, it:

- is not strictly required, so it's a regression risk (sorry to be boring, but regressions are **a nightmare**)
- adds code duplication on more than one level, so I couldn't accept it as-is anyway

We need a path forward, and, as I see it, the easiest way is just dropping this block.